<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>384</y>
      <width>511</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="realizedOutcomeLabel_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>ID Fournisseur :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ssnLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Adresse :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="monthlyCompLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Numero :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hourlyCompLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Email :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="doneHoursLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Produit :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="doneHoursLabel_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Date :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="doneHoursLabel_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Conditions :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="doneHoursLabel_4">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Evaluations :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="id">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nom"/>
         </item>
         <item>
          <widget class="QLineEdit" name="adresse"/>
         </item>
         <item>
          <widget class="QLineEdit" name="numero"/>
         </item>
         <item>
          <widget class="QLineEdit" name="email"/>
         </item>
         <item>
          <widget class="QLineEdit" name="produit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="date">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="conditions">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="evaluations">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>751</x>
      <y>334</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Liste des fournisseur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>660</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>511</width>
      <height>251</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>58</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/</string>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trier">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>340</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>570</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditrecherche">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modif">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>660</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="lineEdit_supp">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>570</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="affichage">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>431</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afficher">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>afficher</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>90</red>
          <green>116</green>
          <blue>82</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gestion Fournisseur&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rechercher">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);</string>
    </property>
    <property name="text">
     <string>rechercher</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewrecherche">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>340</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 116, 82);
 font: 87 9pt &quot;Arial Black&quot;; 
  border-radius: 10px ; /* Arrondi dans tous les coins */
  border: 0px solid rgb(151, 181, 163); /* Bordure de la QLabel en couleur RGB spécifiée */   /*background-color: transparent;*/
border-color:rgb(155,181,163);
</string>
    </property>
    <property name="text">
     <string>pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>stat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CHATBOT</string>
    </property>
   </widget>
   <zorder>tableView_5</zorder>
   <zorder>tableView_2</zorder>
   <zorder>afficher</zorder>
   <zorder>tableView_4</zorder>
   <zorder>widget</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_trier</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>lineEditrecherche</zorder>
   <zorder>pushButton_modif</zorder>
   <zorder>lineEdit_supp</zorder>
   <zorder>label_7</zorder>
   <zorder>rechercher</zorder>
   <zorder>tableViewrecherche</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_stat</zorder>
   <zorder>affichage</zorder>
   <zorder>pushButton_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
